ci/test/00_setup_env_native_valgrind.sh

```diff
@@ -5,11 +5,11 @@
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
 export LC_ALL=C.UTF-8
-
+export TEST_RUNNER_EXTRA="--exclude feature_block,rpc_bind,feature_abortnode,wallet_createwallet"
```

Comment:
```suggestion
export TEST_RUNNER_EXTRA="--exclude feature_block,rpc_bind,feature_abortnode,wallet_createwallet,wallet_multiwallet"
```

Maybe exclude this for now due to the failure  https://travis-ci.org/bitcoin/bitcoin/jobs/636719655#L4607